### PR TITLE
Dropdown display users issue

### LINKED ISSUE #145: show all registered and active users in Team Leader and Team Member drop down
right now only employee role is shown in both dropdowns. It should display all active users

### PR DESCRIPTION
**Pull request by iamronakshah**
fixes #145 

Changed File :
UserRepository.cs : Updated GetAllEmployees method in UserRepository.cs

### CODE REVIEW

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs [hunk 1]
```diff
@@ -227,18 +228,14 @@ public IEnumerable<UserAc> GetAllUsers()
         /// <returns>List of all Employees</returns>
         public async Task<List<UserAc>> GetAllEmployees()
         {
-            var employees = await _userManager.GetUsersInRoleAsync(_stringConstant.RoleEmployee);
+            var users =await _userManager.Users.Where(user=>user.IsActive==true).ToListAsync() ;
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs]: we never compare boolean. IsActive itself will return true or false. There is no need for comparison.

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs [hunk 2]
```diff
@@ -227,18 +228,14 @@ public IEnumerable<UserAc> GetAllUsers()
...
             var userList = new List<UserAc>();
-
-            foreach (var user in employees)
+            foreach (var user in users)
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs]: There is no need for a foreach loop here. Automapper has feature that will convert entire collection too.

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs [hunk 3]
```diff
@@ -227,18 +228,14 @@ public IEnumerable<UserAc> GetAllUsers()
...
             {
-
                 var listItem = _mapperContext.Map<ApplicationUser, UserAc>(user);
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs]: listItem is wrong naming convention. it should have been userAc.

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs [hunk 4]
```diff
@@ -227,18 +228,9 @@ public IEnumerable<UserAc> GetAllUsers()
...
-            {
-
-                var listItem = _mapperContext.Map<ApplicationUser, UserAc>(user);
-
-                userList.Add(listItem);
-            }
-
-            return userList.OrderBy(users => users.FirstName).ToList();
+            var users =await _userManager.Users.Where(user=>user.IsActive).ToListAsync() ;
+            var userAc= _mapperContext.Map<List<ApplicationUser>,List<UserAc>>(users);
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs]: naming convention is wrong. now it's a collection and not a single item. userAclist or userAcCollection would be proper name.

Comment (iamronakshah) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs]: I have updated my naming convention userAc to userAclist.

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs [hunk 5]
```diff
@@ -227,18 +228,9 @@ public IEnumerable<UserAc> GetAllUsers()
...
-            foreach (var user in employees)
-            {
-
-                var listItem = _mapperContext.Map<ApplicationUser, UserAc>(user);
-
-                userList.Add(listItem);
-            }
-
-            return userList.OrderBy(users => users.FirstName).ToList();
+            var users =await _userManager.Users.Where(user=>user.IsActive).ToListAsync() ;
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs]: Why can't you do an OrderBy directly after Where clause? this way method would just be of 2 lines.

Comment (iamronakshah) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs]: I have updated my code now it just 2 line code.

**Pull request merged**